package.json

```diff
@@ -117,6 +117,7 @@
     "@types/node": "16.11.39",
     "@types/node-fetch": "2.5.12",
     "@types/node-forge": "1.3.1",
+    "@types/plist": "^3.0.2",
```

Comment:
We're now pinning versions. Should be `"3.0.2"`

Reply:
Good catch.  changed.